lib/Doctrine/Common/Persistence/AbstractManagerRegistry.php

```diff
@@ -1,56 +1,42 @@
 <?php
 namespace Doctrine\Common\Persistence;
 
+use function explode;
+use function sprintf;
+use function strpos;
+
 /**
  * Abstract implementation of the ManagerRegistry contract.
  *
  * @link   www.doctrine-project.org
```

Comment:
We can safely remove all `@link   www.doctrine-project.org`.
(This annotation isn't forbidden by our CS as it may contain useful information so it needs to be done manually - one nice regexp :) .)